src/nvim/buffer_defs.h

```diff
@@ -525,6 +525,7 @@ struct file_buffer {
   int b_flags;                  // various BF_ flags
   int b_locked;                 // Buffer is being closed or referenced, don't
                                 // let autocommands wipe it out.
+  int b_ro_locked;              // Replacement for curbuf_lock
```

Comment:
meaningless as `curbuf_lock` no longer exists, instead something like the removed comment there  `/// Non-zero when the buffer can't be changed.  Used for FileChangedRO.`

Reply:
Done